using_images/sti_images/python.adoc

```diff
@@ -264,8 +264,13 @@ openshift/python-33-rhel7
 To set these environment variables, you can place them into `.sti/environment`
 file inside your source code repository.
 
-* [envvar]#APP_FILE# - This variable specifies file name (defaults to `app.py`) passed to the python interpreter which is responsible for launching application.
+* `*APP_FILE*` - This variable specifies file name (defaults to *_app.py_*) passed to the python interpreter which is responsible for launching the application.
 
-* [envvar]#APP_MODULE# - This variable specifies WSGI callable. It is of the pattern `$(MODULE_NAME):$(VARIABLE_NAME)`, where module name is a full dotted path and the variable name refers to a inside the specified module. If using `setup.py` for installing the application the module name can be read from that file and variable will default to `application`, eg. see https://github.com/openshift/sti-python/tree/master/3.3/test/setup-test-app[setup-test-app].
+* `*APP_MODULE*` - This variable specifies the WSGI callable. It follows the
```

Comment:
@soltysh -- added "the" here in the sentence mentioning WSGI callable, but unfamiliar with this term. Is that correct? The callable is a noun, so it needs "the"?